pints/_mcmc/_adaptive_covariance.py

```diff
@@ -168,25 +174,42 @@ def tell(self, fx):
         # Adapt covariance matrix
         if self._adaptive:
             # Set gamma based on number of adaptive iterations
-            gamma = self._adaptations ** -0.6
+            self._gamma = self._adaptations ** -self._eta
             self._adaptations += 1
 
-            # Update mu, log acceptance rate, and covariance matrix
-            self._mu = (1 - gamma) * self._mu + gamma * self._current
-            self._loga += gamma * (accepted - self._target_acceptance)
-            dsigm = np.reshape(self._current - self._mu, (self._dimension, 1))
-            self._sigma = (
-                (1 - gamma) * self._sigma + gamma * np.dot(dsigm, dsigm.T))
+            # For localised AM
+            if self._localised:
+                self._fit_gaussian_mixture()
+                self._initial_fit = False
```

Comment:
Needs a comment to explain what this is. And should be defined in `__init__` if it's used...